runelite-client/src/main/java/net/runelite/client/plugins/questlist/QuestListPlugin.java

```diff
@@ -170,9 +170,16 @@ private void addQuestButtons()
 	@Subscribe
 	public void onVarbitChanged(VarbitChanged varbitChanged)
 	{
-		if (isChatboxOpen() && !isOnQuestTab())
+		if (!isOnQuestTab())
 		{
-			chatboxPanelManager.close();
+			updateFilter("");
+			questSearchButton.setOnOpListener((JavaScriptCallback) e -> openSearch());
+			questSearchButton.setAction(1, MENU_OPEN);
+
+			if (isChatboxOpen())
+			{
+				chatboxPanelManager.close();
+			}
```

Comment:
This code should be de-duplicated as it is currently completely copied in both the varbit and varclientint subscribers.

Reply:
👍